account_invoice_line_salesperson/models/account_invoice_line.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountInvoiceLine(models.Model):
+    _inherit = "account.invoice.line"
+
+    user_id = fields.Many2one(
+        "res.users", "Sales Person", readonly=True, compute="_compute_user_id",
+    )
```

Comment:
```suggestion
    salesman_id = fields.Many2one(
        "res.users", "Sales Person", compute="_compute_salesman_id",
    )
```
No need to have `readonly=True` which is the default for computed fields.

Reply:
@TomohiroAbe-Quartile On second thought, this field needs to be stored if it's to be used in a SQL view.  In that case, having `readonly=True` is valid, but you need to add `@depends()` to the compute method.